docs/x509/reference.rst

```diff
@@ -3217,6 +3228,15 @@ instances. The following common OIDs are available as constants.
 
         Corresponds to the dotted string ``"1.3.6.1.5.5.7.48.1.2"``.
 
+
+.. class:: AttributeOID
+
+    .. versionadded:: 3.0
+
+    .. attribute:: CHALLENGE_PASSWORD
+
+        Corresponds to the dotted string ``"1.2.840.113549.1.9.7"``.
```

Comment:
Note: I intend to add a doctest here in a subsequent PR, but I'm going to replace the existing CSR which will necessitate changing a few other doctest results so I wanted to keep that until after this lands.